libjsengine/JSV8Engine.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file JSV8Engine.cpp
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ * Ethereum client.
+ */
+
+#include <memory>
+#include "JSV8Engine.h"
+#include "libjsengine/JSEngineResources.hpp"
+
+using namespace std;
+using namespace dev;
+using namespace dev::eth;
+
+namespace dev
+{
+namespace eth
+{
+
+static char const* toCString(v8::String::Utf8Value const& _value)
+{
+	if (*_value)
+		return *_value;
+	throw JSPrintException();
+}
+
+// from:        https://github.com/v8/v8-git-mirror/blob/master/samples/shell.cc
+// v3.15 from:  https://chromium.googlesource.com/v8/v8.git/+/3.14.5.9/samples/shell.cc
+void reportException(v8::TryCatch*_try_catch)
```

Comment:
bad style.

should be `_tryCatch`

will stop here - check style for rest carefully before changing to `looksgood`.